static/css/com/nav/global-nav.scss

```diff
@@ -180,8 +207,74 @@
     height: 20px;
     width: 20px;
     background: url('../layout/images/icon_cross.svg') center center no-repeat;
+
     &:hover {
       transform: scale(0.9, 0.9);
     }
   }
+}
+
+.solutions-menu {
+  width: 260px;
+  border: 1px solid rgba(255, 255, 255, 0.2);
+  background-color: #27282C;
+  position: absolute;
+  top: 70px;
```

Comment:
It's looks like hard dependency on menu size, may be it's better to use `bottom`?

Reply:
It makes sense, fixed it, now use `bottom`.

Reply:
use variables in this place and header height looks better, but it's ok too